civihr_default_theme/scss/components/_leave-calendar.scss

```diff
@@ -0,0 +1,122 @@
+$chr-leave-calendar-cell-height:                  35px;
+$chr-leave-calendar-cell-font-size:               10px;
+$chr-leave-calendar-item-requested-border-color:  #c7cbd0;
+$chr-leave-calendar-item-requested-border-height: 0.5em;
+$chr-leave-calendar-legend-size:                  20px;
+
+.chr_leave-calendar {
+
+  @media (min-width: $screen-lg-min) {
+    table-layout: fixed;
+  }
+}
+
+
+.chr_leave-calendar__days {
+
+  > th {
+    font-size: 10px;
+    padding: 10px !important;
+    text-align: center;
+
+    @media (min-width: $screen-lg-min) {
+      padding: 10px 0 !important;
+    }
+
+    > span {
+      display: block;
+    }
+  }
+}
+
+
+.chr_leave-calendar__items {
+
+  > td {
+    overflow: hidden;
+    height: $chr-leave-calendar-cell-height;
+    padding: 0 !important;
+  }
+}
+
+
+.chr_leave-calendar__item {
+  color: $light;
+  display: block;
+  font-size: $chr-leave-calendar-cell-font-size;
+  height: 100%;
+  position: relative;
+  text-align: center;
+  text-transform: uppercase;
+  width: 100%;
+
+  > span {
+    bottom: 0;
+    height: 1.2em;
```

Comment:
Please move `top`, `left`, `bottom`, `right` properties together. 
If the `top`, `left`, `bottom`, `right`, `position`, 'margin' is used to center an element, better create a mixin/class